src/bindings/overlay.js

```diff
@@ -130,7 +141,10 @@ function overlay(sourceElement, translationElement) {
   // cleared if a new language doesn't use them; https://bugzil.la/922577
   if (translationElement.attributes) {
     for (k = 0, attr; (attr = translationElement.attributes[k]); k++) {
-      if (isAttrAllowed(attr, sourceElement)) {
+      if (isAttrAllowed({
+          ns: DOM_NAMESPACES[translationElement.namespaceURI],
+          name: attr.name
+        }, sourceElement)) {
         sourceElement.setAttribute(attr.name, attr.value);
```

Comment:
We might want to start using `setAttributeNS` as well as `attr.localName` throughout this file.  We can do it in a follow up.